lib/openbook.ts

```diff
@@ -1,40 +1,45 @@
-import { Program, BN } from '@coral-xyz/anchor';
```

Comment:
We should be using the client, unless there's something we need to do here we can't do with the client.